src/textual/widget.py

```diff
@@ -2080,6 +2098,7 @@ def get_pseudo_classes(self) -> Iterable[str]:
             Names of the pseudo classes.
 
         """
+        yield "disabled" if self._self_or_ancestors_disabled else "enabled"
```

Comment:
For the sake of micro-optimization I would unroll this code. I.e. make fewer implicit function calls.

Reply:
What does this mean? Turn the conditional expression into the `if` below?

```py
if self._self_or_ancestors_disabled:
    yield "disabled"
else:
    yield "enabled"
```

If so, why is that a micro-optimisation?

Reply:
By unrolling I mean copy the code from `self._self_or_ancestors_disabled` to here rather than the property, and repeat so that there are fewer calls.

Function calls in Python are relatively expensive, and this method is called a lot.

Reply:
Thanks for explaining.